### ISSUE TITLE
"Game" in "Game or Player?" slides in whenever you open the Report menu.

### ISSUE THREAD

**Issue by boynedmaster** [Roblox/Core-Scripts#1018]
A minor glitch that I've noticed. The text "Game" will always slide in when you click Report rather than it not animating. 

https://i.gyazo.com/3c069451ef7212d9d64e82a4a6d3a01b.mp4

**Comment by VRService** [comment]
Yes, I also noticed this. I can confirm that it shouldn't animate each time.

**Comment by cliffchapmanrbx** [comment]
Hi there, thank you for your feedback. We're going to be closing this repository as we've chosed [the Roblox Dev Forum](https://devforum.roblox.com/) as our venue for developer feedback. Please feel free to migrate your feedback to a DevForum post if you feel it is still relevant.
